about-osm-community/_posts/0000-01-01-history-of-osm.md

```diff
@@ -4,3 +4,9 @@ title: What is the history of OSM?
 
 
 OpenStreetMap was founded in the UK in 2004 by Steve Coast, and taken up by a community of map enthusiasts who wanted a free and open source of data. It has since grown to more than 1 million contributors around the world: individual volunteers, map enthusiast, GIS and geography data specialists, companies that use map data, humanitarian organizations, government agencies and more.
+
+OpenStreetMap embarqued in 2008 for the transformation from a hobbyist project to a sustainble non-government organization. On important step has been completed in 2012: the data had before been licensed jointly by many many individuals under CC-BY-SA 2.0. Now it is licensed under the (Open Database License 1.0)[https://wiki.osmfoundation.org/wiki/Licence] by the (OpenStreetMap Foundation)[https://wiki.osmfoundation.org/wiki/Main_Page], abbreviated OSMF, that has been founded (for this very pupose)[https://blog.openstreetmap.org/about/].
+
+Since then, OpenStreetMap has become incoporated (at local level)[https://wiki.openstreetmap.org/wiki/Foundation/Local_Chapters] as well. And other communication channels have been added: You can even (discuss individual changes)[https://blog.openstreetmap.org/2014/11/02/introducing-changeset-discussions/].
```

Comment:
also handled in this section as well: https://welcome.openstreetmap.org/about-osm-community/local-chapters/